apps/ideaspace/src/components/modules/SubmissionForm/SubmissionForm.js

```diff
@@ -63,7 +75,10 @@ function SubmissionForm() {
     values['features'] = values['features'].trim();
     values['experience'] = values['experience'].trim();
     values['extraInfo'] = values['extraInfo'].trim();
-    values['involveLevel'] = values['involveLevel'].trim()
+    const involveValueForDataBase = getInvolveLevel(
```

Comment:
Nit: maybe the function can be called `getDBInvolveLevel`.

Reply:
Changed this, thanks!